app/value_objects/conviction_type.rb

```diff
@@ -57,14 +57,14 @@ def initialize(raw_value, params = {})
     ABSOLUTE_DISCHARGE                 = new(:absolute_discharge,               parent: DISCHARGE, skip_length: true, calculator_class: Calculators::ImmediatelyCalculator),
     CONDITIONAL_DISCHARGE              = new(:conditional_discharge,            parent: DISCHARGE, relevant_order: true, calculator_class: Calculators::AdditionCalculator::PlusZeroMonths),
 
-    FINE                               = new(:fine,                             parent: FINANCIAL, skip_length: true, calculator_class: Calculators::AdditionCalculator::StartPlusSixMonths),
-    COMPENSATION_TO_A_VICTIM           = new(:compensation_to_a_victim,         parent: FINANCIAL, relevant_order: true, calculator_class: Calculators::CompensationCalculator),
+    FINE                               = new(:fine,                             parent: FINANCIAL, no_drag_through: false, skip_length: true, calculator_class: Calculators::AdditionCalculator::StartPlusSixMonths),
```

Comment:
Which ticket does this change relate to?

Reply:
https://dsdmoj.atlassian.net/browse/CDPT-1860?